src/apprt/gtk/ghostty_gtk_compat.h

```diff
@@ -12,3 +12,11 @@
 #ifndef G_APPLICATION_DEFAULT_FLAGS
 #define G_APPLICATION_DEFAULT_FLAGS G_APPLICATION_FLAGS_NONE
 #endif
+
+#ifndef False
+#define False 0
+#endif
+
+#ifndef True
+#define True 1
+#endif
```

Comment:
Why is this necessary?

Reply:
Somewhere in `<gdk/x11/gdkx.h>` or `<X11/XKBlib.h>` C macros for `True` and `False` get defined and they are used in a couple of places in `gtk/Window.zig` and `gtk/Surface.zig`. I suppose it would be just as easy to replace the uses in the Zig code.

Reply:
Yeah I would recommend that